src/nvim/testdir/runnvim.sh

```diff
@@ -16,13 +16,15 @@ main() {(
   export NVIM_TEST_ARGC=$#
   local arg
   local i=0
+  # shellcheck disable=SC2034
```

Comment:
"unused $arg" (used in eval)

Reply:
can it go next to the `do`?

      for arg ; do  # shellcheck disable=SC2034

That is mildly useful for avoiding comments getting "detached" from the code, especially since it is not clear what this shellcheck comment is referring to.

Reply:
No, but we can have a comment:

> # shellcheck disable=SC2034  # (unused "arg", used in "eval").